tools/binlog/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: TiDB-Binlog Cluster简介
+category: tools
+aliases: ['/docs-cn/tools/tidb-binlog-cluster/']
+---
+
+# TiDB-Binlog Cluster 简介
+
+本文档介绍 cluster 版本 TiDB-Binlog 的架构以及部署方案。
```

Comment:
@GregoryIan If we don't want to show the other two versions, then it would be unnecessary to mention the `cluster` version here. I think we can remove this sentence. What do you think?

Reply:
It’s ok

Reply:
@ericsyh Then please help remove it. Thanks!

Reply:
Updated in https://github.com/pingcap/docs-cn/pull/1264/commits/79029b7438517734a3a0101a3c367b8a7c0ccc43